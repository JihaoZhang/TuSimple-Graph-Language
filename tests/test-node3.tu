int main()
{	
	node @{bool} n1;
	node @{bool} n2;

	new n1;
	new n2;
	if(n1.value()) print(1);
	else print(0);
	if(n2.value()) print(1);
	else print(0);


	prints(n1.name());
	prints(n2.name());

	return 0;
}