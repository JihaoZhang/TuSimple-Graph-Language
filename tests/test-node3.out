0
0
n1
n2